packages/react/src/components/VideoPlayer/VideoPlayer.js

```diff
@@ -32,10 +41,23 @@ const VideoPlayer = ({ videoId, showDescription }) => {
     })();
   }, [videoId, videoPlayerId]);
 
+  const classnames = cx(
+    `${prefix}--video-player`,
+    { [`${prefix}--video-player--inverse`]: inverse },
+    customClassName
+  );
+  const textclass = cx(
+    `${prefix}--video-player__video-description`,
+    {
+      [`${prefix}--video-player__video-description--inverse`]: inverse,
+    },
+    customClassName
```

Comment:
I don't think we need to add inverse modifier for the text. We can just use the 

`.#{$prefix}--video-player--inverse .#${prefix}--video-player__video-description {}` selector instead in the css

Reply:
updating